### DISCUSSION TITLE
Possible Solution to, "Windows could not set the offline locale information.  Error code: 0x80FF0000"

### DISCUSSION THREAD

**Discussion by Geoppolis** [memstechtips/UnattendedWinstall#209] [General]
Greetings,

Bear with me, this may take some typing.

I just built a brand new, factory-fresh PC with the following parts:
AMD Ryzen 7 7800X3D CPU
Asus B650-E WIFI motherboard
Samsung 990Pro PCIe NVMe M.2 SSD (1TB)
G.Skill Flare X5 DDR5 2x16GB RAM

I've spent 2 days trying to get Windows 10 Home installed using "Method 1" (Memory's WMUtil method) with UnattendedWinstall version 2.1.0. During each attempt, as soon as the "Getting files ready for installation" step (step 2) would reach 100%, I would get an error box with the following message:

"Windows could not set the offline locale information.  Error code: 0x80FF0000"

Information on this error online is stochastic and unfocused.  The general belief seems to be that this is a hardware error.  However, my NVMe drive and controller tested good (via the Asus BIOS), and my ram also tested good (via memtest86).  This furthermore being brand new hardware, it seems likely that it's good.

I tried all the following install methods, multiple times:

Memory's "Method 1" (preferred), which uses WMUtil to generate the ISO and Ventoy to deploy a bootable USB
Booting Ventoy in "Normal" mode
Booting Ventoy in "WIMBOOT" mode
Turning BIOS CSM on/off
Turning BIOS legacy boot on/off
Creating the Ventoy USB with MBR partitioning
Creating the Ventoy USB with GPT partitioning

In a final act of desperation, I tried a flavor of Memory's method 2:
1. Download the Windows Media Creation tool
2. Select "create a bootable USB" within the media creation tool
3. Copy Memory's autounattend.xml file and UWScript.ps1 to the USB (I don't think UWScript is necessary, but I copied it anyway)

Now - here's the strange part: Microsoft recommends having no peripherals except a keyboard and mouse connected to your system during install.  I was using a Logitech wireless mouse the entire time for my install, and, on a whim, I decided to replace it with a normal wired mouse.  IT WORKED.

Windows installed without error, the autounattend script ran flawlessly, and I have a clean windows install exactly as expected for  UnattendedWinstall V2.

To test whether it was the wireless mouse causing my issue, I went back and attempted to repeat "Method 1" with my wired mouse attached, and the install failed with exactly the same 0x80FF0000 error code as before.  Only the USB drive with the MS Media Creation Tool-created ISO (and autounattend.xml) worked.  (Implying the mouse change was immaterial.)

I know this tale is bizarre, but I wanted to post it in case it helps some other poor soul struggling with this cryptic error.

On edit:
I reflecting on this issue, I want to say I suspect it had little or nothing to do with changing from a wireless to a wired mouse.  I think what I ran into was some sort of insidious issue with the Method 1 process.  Because it was only the MS Media Creation Tool-created USB drive (with autounattend.xml in the root) that successfully avoided the 0x80FF0000 error.

**Comment by pseudokoder** [comment]
Hi Geoppolis!

Thank you writing this. I too am having the same Error: 0x80FF0000 during installation. Brand new PC build:

MB: MSI B650M Project Zero AM5
CPU: AMD Ryzen 7 7700X
RAM: Team Group T-CREATE EXPERT 32GB (2 x 16GB) 288-Pin PC RAM DDR5 6000
SSD: Team Group MP44 M.2 2280 2TB PCIe 4.0 x4 with NVMe

For me this is day one of installing the customized Win10Pro OS. I was running an UN-activated version of Win 10 Pro on a slightly older SSD so I could test all the other hardware while waiting for my new 2TB SSD to arrive. Everything worked great with the default OEM Win 10 Pro installation.

I am using the same USB key with MS Media Creation Tool-created ISO except I just downloaded the autounattend.xml in the root and that is when I'm getting that error.

Just for kicks, I disconnected my keyboard and mouse from my USB hub and plugged them both directly into the back I/O panel. I came across your error submission here and decided to post this while I attempt to troubleshoot further. It appears that I'm already using the same method that you said you used to fix your error so I'm not sure what I'm going to try next.

Only difference is I didn't copy over the UWScript.ps1.  But I don't think that should affect this error.

On edit:
I removed the x86 blocks from the xml file, and it appears to be working now.  That's all I did!  Hope this helps someone else.

> Greetings,
> 
> Bear with me, this may take some typing.
> 
> I just built a brand new, factory-fresh PC with the following parts: AMD Ryzen 7 7800X3D CPU Asus B650-E WIFI motherboard Samsung 990Pro PCIe NVMe M.2 SSD (1TB) G.Skill Flare X5 DDR5 2x16GB RAM
> 
> I've spent 2 days trying to get Windows 10 Home installed using "Method 1" (Memory's WMUtil method) with UnattendedWinstall version 2.1.0. During each attempt, as soon as the "Getting files ready for installation" step (step 2) would reach 100%, I would get an error box with the following message:
> 
> "Windows could not set the offline locale information. Error code: 0x80FF0000"
> 
> Information on this error online is stochastic and unfocused. The general belief seems to be that this is a hardware error. However, my NVMe drive and controller tested good (via the Asus BIOS), and my ram also tested good (via memtest86). This furthermore being brand new hardware, it seems likely that it's good.
> 
> I tried all the following install methods, multiple times:
> 
> Memory's "Method 1" (preferred), which uses WMUtil to generate the ISO and Ventoy to deploy a bootable USB Booting Ventoy in "Normal" mode Booting Ventoy in "WIMBOOT" mode Turning BIOS CSM on/off Turning BIOS legacy boot on/off Creating the Ventoy USB with MBR partitioning Creating the Ventoy USB with GPT partitioning
> 
> In a final act of desperation, I tried a flavor of Memory's method 2:
> 
>     1. Download the Windows Media Creation tool
> 
>     2. Select "create a bootable USB" within the media creation tool
> 
>     3. Copy Memory's autounattend.xml file and UWScript.ps1 to the USB (I don't think UWScript is necessary, but I copied it anyway)
> 
> 
> Now - here's the strange part: Microsoft recommends having no peripherals except a keyboard and mouse connected to your system during install. I was using a Logitech wireless mouse the entire time for my install, and, on a whim, I decided to replace it with a normal wired mouse. IT WORKED.
> 
> Windows installed without error, the autounattend script ran flawlessly, and I have a clean windows install exactly as expected for UnattendedWinstall V2.
> 
> To test whether it was the wireless mouse causing my issue, I went back and attempted to repeat "Method 1" with my wired mouse attached, and the install failed with exactly the same 0x80FF0000 error code as before. Only the USB drive with the MS Media Creation Tool-created ISO (and autounattend.xml) worked. (Implying the mouse change was immaterial.)
> 
> I know this tale is bizarre, but I wanted to post it in case it helps some other poor soul struggling with this cryptic error.
> 
> On edit: I reflecting on this issue, I want to say I suspect it had little or nothing to do with changing from a wireless to a wired mouse. I think what I ran into was some sort of insidious issue with the Method 1 process. Because it was only the MS Media Creation Tool-created USB drive (with autounattend.xml in the root) that successfully avoided the 0x80FF0000 error.

**Reply by Geoppolis** [reply]
Thank you, pseudokoder, for that!  I'm glad someone else saw this and found it useful.

**Reply by sylvi-3** [reply]
I am absolutely floored.
I was having this exact same issue, and unplugging my Logitech G502 Hero reciever (as well as my microphone and a USB hub that has a Logitech webcam attached) fixed it. I did not change anything else in the process.
I have absolutely no idea how this could be the cause of my issue, but it was! Thank you!
